hnn_core/batch_simulate.py

```diff
@@ -0,0 +1,157 @@
+"""Batch simulation."""
+
+# Authors: Abdul Samad Siddiqui <abdulsamadsid1@gmail.com>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+#          Ryan Thorpe <ryan_thorpe@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+from joblib import Parallel, delayed
+from hnn_core import simulate_dipole
+from hnn_core.network_models import (jones_2009_model,
+                                     calcium_model, law_2021_model)
+
+
+class BatchSimulate:
+    def __init__(self, set_params, net_name='jones', tstop=170,
+                 dt=0.025, n_trials=1):
```

Comment:
copy parameters and docstrings from `simulate_dipole` and pass options to `_run_single_sim()`